contribute/design/proposal-cstor-orchestration.md

```diff
@@ -0,0 +1,57 @@
+# Managing cStor based OpenEBS Volumes. 
+
+## Background 
+
+
+## Design
+
+Provisioning of cStor based Volume involve two steps. 
+
+1. Creating cStor StoragePools:
+
+   * Admin will create SPC that contains information like:
+     - type=block
+     - name
+     - disks list (type of PDC or specific disks)
+     - resource requests (optional, like CPU, RAM)
+     - raid type (optional, default _stripe_)
+     - nodes list (optional, default: _all_)
+     - max-pools (optional, default: _none_)
```

Comment:
Will count be a better name? Will there be a case of min-pools?

Reply:
max-pools was used to set an upper limit. there is no lower limit.